src/views/bootsources/components/BootableVolumesRow.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { NO_DATA_DASH } from '@kubevirt-utils/resources/vm/utils/constants';
+import {
+  K8sGroupVersionKind,
+  ResourceLink,
+  RowProps,
+  TableData,
+  Timestamp,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+import { getDataSourcePreferenceLabelValue, usePreferenceReadableOS } from '../utils/utils';
+
+const BootableVolumesRow: React.FC<
+  RowProps<V1beta1DataSource, { groupVersionKind: K8sGroupVersionKind }>
+> = ({ obj, activeColumnIDs, rowData: { groupVersionKind } }) => {
+  return (
+    <>
+      <TableData id="name" activeColumnIDs={activeColumnIDs}>
+        <ResourceLink groupVersionKind={groupVersionKind} name={obj.metadata.name} />
+      </TableData>
+      <TableData id="os" activeColumnIDs={activeColumnIDs}>
+        {usePreferenceReadableOS(obj)}
+      </TableData>
+      <TableData id="description" activeColumnIDs={activeColumnIDs}>
+        {obj?.metadata?.annotations?.description || NO_DATA_DASH}
+      </TableData>
+      <TableData id="preference" activeColumnIDs={activeColumnIDs}>
+        {getDataSourcePreferenceLabelValue(obj)}
+      </TableData>
+      <TableData id="namespace" activeColumnIDs={activeColumnIDs}>
+        <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
```

Comment:
can you replace kind with groupVersionKind?

Reply:
I pretty know  that "kind" prop is deprecated for `ResourceLink`.  Tried not to use that in here because of that, but I wasn't successful. Seems that this is not working with "Namespace" as expected. IMHO that also explains why we still use "kind" in all other places where we display this kind of the resource.